test/doc/algo/convert.cpp

```diff
@@ -0,0 +1,31 @@
+#include <eve/module/core.hpp>
+#include <eve/algo.hpp>
+#include <iostream>
+#include <vector>
+#include <eve/views/iota.hpp>
+#include <eve/views/reverse.hpp>
+#include "detail/print.hpp"
+
+int main()
+{
+  std::vector<int> v(10);
+  eve::algo::iota(v, 1);
+  auto v_as_doubles = eve::views::convert(v, eve::as<double>{});
+  std::vector<float> vm(10);
+  eve::algo::transform_to(v_as_doubles, vm, [](auto i){return i*20.3; });
+  auto v_as_int8 = eve::views::convert(vm, eve::as<std::int8_t>{});
+
+
+  std::cout << " <- std::vector<int> v(10);\n";
+  std::cout << " -> auto v_as_doubles = eve::views::convert(v, eve::as<double>{});\n";
+  std::cout << " -> std::vector<float> vm(10);\n";
+  std::cout << " <- eve::algo::transform_to(v_as_doubles, vm, [](auto i){return i*20.3; });\n";
```

Comment:
You should probaly show that `eve::element_type_t<decltype(i)>` is double

Smth like:

[]<typename N>(eve::wide<double, N>)

Reply:
I do not understand.
Where do I put this ?

Reply:
Here:
`[](eve::wide<double, N> i){return i*20.3; }`